docs/source/gbd2019_models/causes/pad/index.rst

```diff
@@ -23,21 +23,56 @@ Peripheral Arterial Disease
   * - PAD
     - Peripheral Arterial Disease
     - 
+  * - ABI
+    - Ankle-Brachial Index
+    - The ankle-brachial index test compares the blood pressure measured at your ankle with the blood pressure measured at your arm. It is a method of checking for PAD.
+
+[ABI-Definition]_  
 
 
 Disease Overview
 ----------------
-Patients with peripheral arterial disease (PAD) have decreased lower extremity arterial perfusion which is commonly referred to as “poor circulation.” In most cases of PAD, atherosclerotic plaques narrow the arterial flow lumen which restricts blood flow to the distal extremity. Reduced blood flow can cause thigh or calf pain with walking due to temporary ischemia of the leg muscles during exertion. Walking pain from PAD is referred to as intermittent claudication which means “to limp.” Many patients with PAD have either no symptoms or atypical complaints that do not strictly conform to the definition of claudication. Others may develop limb-threatening compromise of blood flow, necessitating emergent surgery. PAD acts as a marker for systemic atherosclerosis. PAD usually involves atherosclerotic disease in the abdominal aorta, iliac, and femoral arteries. The pathophysiology of atherosclerosis involves complex interactions between cholesterol and vascular cells the details. Atherosclerotic plaque builds up slowly on the inside of arteries. In the early stages of PAD, the arteries compensate for the plaque buildup by dilating to preserve flow through the vessel. Eventually, the artery cannot dilate any further, and the atherosclerotic plaque starts to narrow the arterial flow lumen.
+Patients with peripheral arterial disease (PAD) have decreased lower extremity arterial perfusion which is commonly referred to as “poor circulation.” In most cases of PAD, atherosclerotic plaques narrow the arterial flow lumen which restricts blood flow to the distal extremity. Reduced blood flow can cause thigh or calf pain with walking due to temporary ischemia of the leg muscles during exertion. Walking pain from PAD is referred to as intermittent claudication which means “to limp.” Many patients with PAD have either no symptoms or atypical complaints that do not strictly conform to the definition of claudication. Others may develop limb-threatening compromise of blood flow, necessitating emergent surgery. PAD acts as a marker for systemic atherosclerosis. PAD usually involves atherosclerotic disease in the abdominal aorta, iliac, and femoral arteries. The pathophysiology of atherosclerosis involves complex interactions between cholesterol and vascular cells. Atherosclerotic plaque builds up slowly on the inside of arteries. In the early stages of PAD, the arteries compensate for the plaque buildup by dilating to preserve flow through the vessel. Eventually, the artery cannot dilate any further, and the atherosclerotic plaque starts to narrow the arterial flow lumen.
```

Comment:
if you use an editor that automatically breaks long lines of text, GitHub might be able to highlight the changes before and after better, which will make it easier for me to review this quickly.

Reply:
Yeah that's a good point. I have been using Sublime text. I could also manually break it into multiple lines.
The difference here is: The pathophysiology of atherosclerosis involves complex interactions between cholesterol and vascular cells **the details.** --> The pathophysiology of atherosclerosis involves complex interactions between cholesterol and vascular cells.